ui/raidboss/common_replacement.js

```diff
@@ -15,13 +15,27 @@ let commonReplacement = {
     ja: 'の封鎖が解かれた',
     ko: '의 봉쇄가 해제되었습니다',
   },
+  '--adds spawn--': {
+    de: '--adds spawn--',
+    fr: '--apparition d\'adds--',
+    ja: '--adds spawn--',
+    cn: '--adds spawn--',
+    ko: '--쫄 소환--',
```

Comment:
@MaikoTan, @Jaehyuk-Lee, @Akurosia, @r4ha.
I need your help translating these lines, if there is any need. They were not translated in the other files, sorry to bother you again.

Reply:
`de: '--Adds erscheinen--'`

Reply:
I am not very sure what is "spawn" meanning.
If it means "appear / turn to visible", I think this can be translate as "--雑魚出現--".
@r4ha what do you think?

Reply:
no problem at kor.

Reply:
> I am not very sure what is "spawn" meanning.
> If it means "appear / turn to visible", I think this can be translate as "--雑魚出現--".
> @r4ha what do you think?

Agree!

Reply:
No one for cn translation? Otherwise I will leave the English words. Thank you all.

Reply:
> No one for cn translation? Otherwise I will leave the English words. Thank you all.

Oh sorry I forgot it!
I think it should be '--小怪出现--' in cn.

Reply:
Thank so much.